py_gtfs_rt_ingestion/ingest.py

```diff
@@ -169,15 +145,15 @@ def lambda_handler(event: LambdaDict, context: LambdaContext) -> None:
     batch files should all be of same ConfigType as each run of this script
     creates a single parquet file.
     """
-    logging.info("Processing event:\n%s", json.dumps(event, indent=2))
-    logging.info("Context:%s", context)
+    logging.info("ingestion_event=%s", json.dumps(event))
+    process_logger = ProcessLogger("ingest_files_lambda")
+    process_logger.log_start()
```

Comment:
I think it would be very helpful to have the `config_type` for each ingest process as part of the metadata being printed